packages/connect-conformance/package-lock.json

```diff
@@ -0,0 +1,71 @@
+{
+  "name": "@connectrpc/connect-node-conformance",
```

Comment:
We shouldn't have a lock file here, there should only be a single one at the repo root. This may have been caused by running npm install in this package.